src/components/tools/geometry-tool/geometry-content.tsx

```diff
@@ -1114,7 +1114,8 @@ export class GeometryContentComponent extends BaseComponent<IProps, IState> {
       }
 
       for (const elt of board.objectsList) {
-        if ((isVisiblePoint(elt) || isVisibleEdge(elt) || isVisibleMovableLine(elt) || isAxisLabel(elt)) &&
+        if ((isVisiblePoint(elt) || isVisibleEdge(elt) || isVisibleMovableLine(elt) || isAxisLabel(elt) ||
+              isComment(elt) || isMovableLineEquation(elt)) &&
```

Comment:
My inclination is that this laundry list has gotten too long and should be replaced with an appropriately named function. The remaining question is what should that function be named? `shouldInterceptPointCreateClicks()`?